tox.ini

```diff
@@ -0,0 +1,31 @@
+[tox]
+envlist = linters
+skipsdist = true
+
+[testenv]
+allowlist_externals=*
+deps =
+       -r{toxinidir}/test_requirements.txt
+commands =
+    {envpython} --version
+
+[testenv:ruff]
+deps = {[testenv:linters]deps}
+commands =
+    bash -c 'ruff check --exclude .tox --select ALL --ignore INP001 .'
+
+[testenv:darglint]
+deps = {[testenv:linters]deps}
+commands =
+    bash -c 'find . -not -path "./.tox/*" -and -name "*.py" -print0 | xargs -0 darglint -s numpy -z full'
+
+[testenv:pylint]
+deps = {[testenv:linters]deps}
+commands = 
+    bash -c 'find . -not -path "./.tox/*" -and -name "*.py" -print0 | xargs -0 pylint --output-format=parseable -sn --disable R0801'
+
+[testenv:linters]
+commands =
+    {[testenv:ruff]commands}
+    {[testenv:darglint]commands}
+    {[testenv:pylint]commands}
```

Comment:
```suggestion
```

Reply:
See above comment about individual envs instead of calling them inside of `linters`.